content/projects/data-science-specific/data-visualisation/mobile-money-viz/_index.md

```diff
@@ -38,15 +38,17 @@ This dataset is the geospatial mapping of all cash outlets in Tanzania in 2012.
 
 ## Instructions:
 
-1. Examine the dataset. Are there any missing observations or columns where the data do not seem valid?
+1. Import the required libraries and load the datadatset [training](training.csv).
+   
+2. Explore the dataset. Are there any missing observations or columns where the data do not seem valid?
 
-2. Get basic descriptive statistics for the dataset.
+3. Get basic descriptive statistics for the dataset.
 
-3. Create appropriate graphs to visually represent the relationship between financial services accessed (non-mobile, mobile, both) and age, gender, marital status, land ownership and type of income.
+4. Create appropriate graphs to visually represent the relationship between financial services accessed (non-mobile, mobile, both) and age, gender, marital status, land ownership and type of income. Provide interpretation for each graph.  
 
-4. What can you conclude about use of financial services in Tanzania so far? Which demographic are associated with mobile money use?
-
-## Variables
+5. What can you conclude about use of financial services in Tanzania so far? Which demographic are associated with mobile money use ?
+   
+## Variables (metadata information)
```

Comment:
This doesn't seem cohesive.

We describe the dataset in the first section, then tell the learner what we need, then describe the dataset the rest of the way right at the end.  Why is stuff seperated out like that? 

A better thing to do would be:

1. introduce the problem 
2. give the learner instructions 
3. tell the learner all about the data

OR

1. introduce the problem 
2. tell the learner all about the data
3. give the learner instructions

Reply:
Sure.